src/app/pages/OnDemandRadioPage/index.test.jsx

```diff
@@ -127,4 +138,70 @@ describe('OnDemand Radio Page ', () => {
       ),
     ).toBeInTheDocument();
   });
+
+  it('should show the audio player on canonical', async () => {
+    const clonedKoreanPageData = clone(koreanPageData);
+    clonedKoreanPageData.content.blocks[0].versions[0].availableUntil = 9999999999999999999999999;
+    const koreanPageDataWithAvailableEpisiode = clonedKoreanPageData;
+    fetch.mockResponse(JSON.stringify(koreanPageDataWithAvailableEpisiode));
+    const { pageData } = await getInitialData('some-ondemand-radio-path');
+    const { container } = render(
+      createAssetPage({ pageData, service: 'korean' }),
+    );
+    const audioPlayerIframeSrc = container
+      .querySelector('iframe')
+      .getAttribute('src');
+
+    expect(audioPlayerIframeSrc).toEqual(
+      'https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_korean_radio/w3cszwcg/ko',
+    );
+  });
+
+  it('should show the audio player on AMP', async () => {
+    const clonedKoreanPageData = clone(koreanPageData);
+    clonedKoreanPageData.content.blocks[0].versions[0].availableUntil = 9999999999999999999999999;
+    const koreanPageDataWithAvailableEpisiode = clonedKoreanPageData;
+    fetch.mockResponse(JSON.stringify(koreanPageDataWithAvailableEpisiode));
+    const { pageData } = await getInitialData('some-ondemand-radio-path');
+    const { container } = render(
+      createAssetPage({ pageData, service: 'korean', isAmp: true }),
+    );
+    const audioPlayerIframeSrc = container
+      .querySelector('amp-iframe')
+      .getAttribute('src');
+
+    expect(audioPlayerIframeSrc).toEqual(
+      'https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_korean_radio/w3cszwcg/ko/amp',
+    );
+  });
+
+  it('should show the expired content message if episode is expired', async () => {
+    const clonedKoreanPageData = clone(koreanPageData);
+    clonedKoreanPageData.content.blocks[0].versions[0].availableUntil = 15856595095250000;
```

Comment:
Cool, I like that we can test this without needing to worry about exact test data.